src/viewers/viewer/globals.js

```diff
@@ -222,7 +222,7 @@ export const PROJECTION = {
  */
 export const UNITS_LENGTH = [
     { unit: 'angstrom',
-      threshold: 0.1, multiplier: 10000, symbol: '\u212B'},
+      threshold: 0.01, multiplier: 10000, symbol: '\u212B'},
```

Comment:
What is the impact of modifying this threshold?

Reply:
If, for example, the minimum length of scalebar was ```0.8 microns```, then we look for a UNIT where the threshold is larger than this (starting at the smallest, ```angstrom```). With the old value of ```0.1```, we pick ```angstrom``` as a suitable unit for a scalebar of ```0.8 microns```. When the scalebar is increased from ```800 angstrom``` to a rounded length, we end up with a length of ```1000 angstrom```.
With the change in threshold, we instead end up picking ```nm``` and we get a scalebar of ```100 nm``` which seems more appropriate.

Reply:
Ah I see, so anything below `0.01 microns` will be turned into angstroms rather than  nanometers. Sounds like a fair cutoff and certainly better than `100 nm`. We can review this and further push towards `1nm` if needed.